rancher-cluster-operations/bulk-components/provider.tf

```diff
@@ -1,7 +1,7 @@
-provider "rancher2" {
-  bootstrap = false
-  api_url   = var.rancher_api_url
-  token_key = var.rancher_token_key
-  insecure  = true
-  timeout   = "300s"
-}
+# provider "rancher2" {
```

Comment:
Do you think you are going to need this in the future?

Reply:
This is leftover from when this was a root module, I'll remove it